packages/webcomponents/src/styles/parts.ts

```diff
@@ -63,15 +63,25 @@ export const card = `card ${text} ${backgroundColor}`;
 
 // Table
 export const table = `table ${text} ${backgroundColor}`;
+export const tableRow = `table-row ${text} ${backgroundColor}`;
 export const tableCell = `table-cell ${text} ${backgroundColor}`;
-export const tableHeadCell = `table-head-cell ${tableCell} ${text} ${backgroundColor}`;
-export const tableCellEven = `table-cell-even ${tableCell} ${text} ${backgroundColor}`;
-export const tableCellOdd = `table-cell-odd ${tableCell} ${text} ${backgroundColor}`;
+export const tableHeadCell = `table-head-cell ${tableCell}`;
+export const tableCellEven = `table-cell-even ${tableCell}`;
+export const tableCellOdd = `table-cell-odd ${tableCell}`;
 export const getAlternateTableCellPart = (index) =>
   index % 2 ? tableCellEven : tableCellOdd;
 export const tableEmpty = `table-empty ${text} ${backgroundColor}`;
 export const tableError = `table-error ${text} ${backgroundColor}`;
 
+// Pagination
+export const paginationItem = `pagination-item ${text} ${backgroundColor}`;
+export const paginationItemText = `pagination-item-text ${text}`;
+export const paginationItemNext = `pagination-item-next ${paginationItem}`;
+export const paginationItemPrevious = `pagination-item-previous ${paginationItem}`;
```

Comment:
From what I can see - I don't believe `paginationItemNext` or `paginationItemPrevious` are being used at all. 

Did you intend to import these and use them in pagination-menu? I noticed `buttonDisabled` and `buttonLink` parts are used instead. 🤔